README.md

```diff
@@ -30,38 +37,158 @@ and uses [lnwallet](https://pkg.go.dev/github.com/lightningnetwork/lnd/lnwallet/
 
 ## Usage Documentation
 
-For a comprehensive guide on how to set up and use Ark, please visit our [Developer Portal](https://arkdev.info).
-For a quick-start with Docker, head over to our [Quick Start guide](https://arkdev.info/docs/quick-start/overview) where you can setup an Ark Server and Clients in a local Bitcoin regtest network.
+Welcome to the `arkd` repository, a Bitcoin server for off-chain Bitcoin transactions using the Ark protocol.
```

Comment:
i'd either drop this or change it to something like "this is what you're going to learn here"